pkg/services/ngalert/api/api.go

```diff
@@ -49,6 +50,10 @@ type Alertmanager interface {
 	TestReceivers(ctx context.Context, c apimodels.TestReceiversConfigBodyParams) (*notifier.TestReceiversResult, error)
 }
 
+type AlertingStore interface {
```

Comment:
Note: All the other interfaces are now gone. This one remains though - we can't do without this one because the API layer talks directly to the store layer. This one is a bit less problematic though, and is consistent with how the MultiOrgAlertmanager tests already work by faking the store.